docs/var.tfvars-doc.md

```diff
@@ -121,6 +121,14 @@ rhcos_image_name    = "<rhcos-image-name>"
 ```
 Note that the boot images should have a minimum disk size of 120GB
 
+These set of variables should be provided when RHCOS should be imported from cloud object storage to your PowerVS service instance
+```
+import_image       = true                           # true/false (default=false)
+bucket_region      = "object_store_bucket_region"   # Region of the Object store bucket
```

Comment:
Below are the regions (8) that I can extract from `ibmlcoud pi sl` (for `ibmcloud_region`)
au-syd, jp-osa, jp-tok, eu-de, eu-gb, ca-tor, br-sao, mon

Below are the bucket regions (9) 
au-syd, jp-osa, jp-tok, eu-de, eu-gb, ca-tor, br-sao, us-south, us-east

From above we have only 7 in common and there could be a possibility of using the image from us-south and us-east. Also user can use region "mon" (montreal) for SI. To avoid the possibility of failures, variable for `bucket_region` was implemented. Hope the reason is justified.

Reply:
If we consider montreal as `ibmcloud_region`, we do not have `bucket_region` for montreal. In this case shall we go ahead with some default region (may be `au_syd`) or fail with error message as "Bucket Region `<bucket_region>` does not exist".

What is your suggestion?

Reply:
all PowerVS regions should be supported, and they are all have responded VPC region. Other VPC regions like `montreal` we don't care since no PowerVS region for it.

Reply:
Implemented this. Now use don't need to provide bucket region.